src/DataManagement/Entries/EntryList.cs

```diff
@@ -1,20 +1,14 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Security.Cryptography.X509Certificates;
+﻿using System.Collections.Generic;
 using System.Text;
-using System.Threading.Tasks;
 using ePiggyWeb.Utilities;
 
-namespace ePiggyWeb.DataManagement
+namespace ePiggyWeb.DataManagement.Entries
 {
-    public class EntryList : List<Entry>
+    public class EntryList : List<IEntry>, IEntryList
     {
         //EntryType is to save up on the amount of times we have to pass the EntryType in methods
         public EntryType EntryType { get; set; }
```

Comment:
Yeah the comment is really unclear, what I meant was that instead of having to add and pass a paramater of EntryType to most of the methods that involve EntryLists, we just have it as a variable inside the EntryList.
Basically just to make methods easier to call